python_modules/dagster/dagster/_core/definitions/resource_definition.py

```diff
@@ -440,5 +440,5 @@ def my_job():
     """
     return ResourceDefinition(
         resource_fn=lambda init_context: init_context.resource_config,
-        config_schema=kwargs or Any,
+        config_schema=kwargs or None,
```

Comment:
`Any` is not a valid config schema, but `None` gets converted to the equivalent of `Any`